dashboards-observability/public/components/common/search/autocomplete_logic.ts

```diff
@@ -51,15 +51,19 @@ import {
   PIPE_COMMA_AFTER_SORT_FIELD,
   PLUS_MINUS_FIELD_IN_FIELDS_LOOP,
   FIELD_AFTER_COMMAND,
-  FIELD_AFTER_STATS_GROUP_BY,
-  FIELD_AFTER_AGGREGATION,
+  FIELD_SPAN_AFTER_GROUP_BY,
+  NUM_FIELD_AFTER_AGGREGATION,
   CLOSE_AFTER_FIELD,
   PIPE_COMMA_BY_AFTER_AGGREGATION,
   PIPE_AFTER_STATS_GROUP_BY,
   AGGREGATION_FOR_STATS,
+  FIELD_AFTER_SPAN,
+  CLOSE_AFTER_SPAN,
+  PIPE_AFTER_SPAN,
 } from '../../../../common/constants/autocomplete';
 
-let currIndex: string = '';
+// let currIndex: string = '';
```

Comment:
can we remove this line if `currIndex` is no longer used?

Reply:
Yes! Good catch.